crypto/openssl/openssl_key_exchange_ecdh.c

```diff
@@ -220,29 +207,29 @@ static bool compute_publicBECDH(ecdh_context_t *key_ex_data)
 	char *hexbuf1 = BN_bn2hex(x);
 
 	LOG(LOG_DEBUG, "Bx %s : bytes %d, %s\n",
-	    BN_is_negative(x) ? "Negative" : "Positive", bn_num_bytes(x),
-	    hexbuf1);
+			BN_is_negative(x) ? "Negative" : "Positive", bn_num_bytes(x),
+			hexbuf1);
 	OPENSSL_free(hexbuf1);
 
 	char *hexbuf2 = BN_bn2hex(y);
 
 	LOG(LOG_DEBUG, "By %s : bytes %d, %s\n",
-	    BN_is_negative(y) ? "Negative" : "Positive", bn_num_bytes(y),
-	    hexbuf2);
+			BN_is_negative(y) ? "Negative" : "Positive", bn_num_bytes(y),
+			hexbuf2);
 	OPENSSL_free(hexbuf2);
 
 	char *hexbuf3 = BN_bn2hex(key_ex_data->_Device_random);
 
 	LOG(LOG_DEBUG, "Device Random  %s : bytes %d, %s\n",
-	    BN_is_negative(key_ex_data->_Device_random) ? "Negative"
-							: "Positive",
-	    bn_num_bytes(key_ex_data->_Device_random), hexbuf3);
+			BN_is_negative(key_ex_data->_Device_random) ? "Negative"
+			: "Positive",
+			bn_num_bytes(key_ex_data->_Device_random), hexbuf3);
 	OPENSSL_free(hexbuf3);
 #endif
 
 	/* 2byte for each blen 3x2 =6 */
 	allocbytes = (bn_num_bytes(x) + bn_num_bytes(y) +
-		      bn_num_bytes(key_ex_data->_Device_random) + 6);
+			bn_num_bytes(key_ex_data->_Device_random) + 6);
```

Comment:
Input validation for expected size ranges is missing

Reply:
size validation happens inside fdo_alloc() function that returns failure if value of allocbytes is greater than a limit.(#define BUFF_SIZE_64K_BYTES 64000)